ExploreASL_ImportConfig.m

```diff
@@ -21,6 +21,13 @@
 % Initialize ExploreASL
 [x] = ExploreASL_Initialize;
 
+% Print warning
+fprintf('\n========================================== WARNING ===========================================\n');
+fprintf(['\nThe support for this ExploreASL functionality will expire with a future release.\n'...
+         'If you want to be able to use this function within a future release, please pull\n'...
```

Comment:
Nice, though not sure if this is sufficiently clear to the user ;) I would add that this is taken over by the BIDS import support straightly through the `ExploreASL_Master` function. Also, what should the user do with the CustomScripts once pulled? You don't need to put a manual here of course :) but it's good if there some minor guidance

Reply:
Good idea, but let's change the wording a bit. `ExploreASL_Master` doesn't create the `imPar`, what `ImportConfig` basically does right now. So we have to say that these settings will be part of the BIDS compliant `sourceStructure.json` file, that can be read using `ExploreASL_Master` directly, right?

Reply:
This should say everything:

![image](https://user-images.githubusercontent.com/46593074/116721128-44149c00-a9dd-11eb-8deb-d0b699f75492.png)

Reply:
Very nice!